salt/digests/config/srv-digests-uwsgi.ini

```diff
@@ -0,0 +1,20 @@
+[uwsgi]
```

Comment:
this is outdated and needs to be in line with https://github.com/elifesciences/digests/blob/develop/config/uwsgi.ini